### PR TITLE
Update import merge tool

### PR DESCRIPTION
**Pull request by benedikt-schesch**

### CODE REVIEW

**Comment by benedikt-schesch** [comment]
@mernst Any clue why the merge tool is crashing in the CI/CD pipeline?

**Comment by mernst** [comment]
OK, I will investigate.

File: src/scripts/merge_tools/merge_plumelib.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# shellcheck disable=SC2016
+git config --local mergetool.merge-plumelib.cmd 'java-merge-tool.sh '"$merge_strategy"' ${BASE} ${LOCAL} ${REMOTE} ${MERGED}'
+git config --local mergetool.merge-plumelib.trustExitCode true
+
+# shellcheck disable=SC2086
+git merge --no-edit $git_strategy "$branch2"
+retVal=$?
+
+# report conflicts
+if [ "$retVal" -ne 0 ]; then
```
[the remark this answers is not in the document]
Comment (benedikt-schesch) [reply src/scripts/merge_tools/merge_plumelib.sh:32]: @mernst I assume it is the the same for the default version

[the remark this answers is not in the document]
Comment (mernst) [reply src/scripts/merge_tools/merge_plumelib.sh]: Yes.  The default version runs annotations, imports, and version-numbers.
(Defaults are listed at https://github.com/plume-lib/merging/#features .)

File: src/scripts/merge_tools/merge_plumelib.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# shellcheck disable=SC2016
+git config --local mergetool.merge-plumelib.cmd 'java-merge-tool.sh '"$merge_strategy"' ${BASE} ${LOCAL} ${REMOTE} ${MERGED}'
+git config --local mergetool.merge-plumelib.trustExitCode true
+
+# shellcheck disable=SC2086
+git merge --no-edit $git_strategy "$branch2"
+retVal=$?
+
+# report conflicts
+yes | git mergetool --tool=merge-plumelib
```
Comment (mernst) [inline src/scripts/merge_tools/merge_plumelib.sh:32]: Any merger that involves "imports" has to be run via `git-mergetool-on-all.sh`.

Comment (benedikt-schesch) [reply src/scripts/merge_tools/merge_plumelib.sh]: @mernst Like this?

Comment (mernst) [reply src/scripts/merge_tools/merge_plumelib.sh]: Yes, that looks right.  Thank you!

File: src/scripts/merge_tools/merge_plumelib.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env sh
+
+# usage: ./gitmerge_ort_imports_ignorespace.sh <clone_dir> <branch-1> <branch-2> <git_strategy> <merge_strategy>
```
Comment (mernst) [inline src/scripts/merge_tools/merge_plumelib.sh:3]: Wrong usage message, it refers to `gitmerge_ort_imports_ignorespace.sh`.

File: src/scripts/merge_tools/merge_plumelib.sh [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+clone_dir=$1
+branch1=$2
+branch2=$3
+git_strategy=$4 #"-Xignore-space-change"
+merge_strategy=$5 #"--only-adjacent"
+
+echo "git_merge_plumelib: Merging $branch1 and $branch2 with git_strategy=$git_strategy and merge_strategy=$merge_strategy"
+
+# shellcheck disable=SC2153 # "JAVA17_HOME is not a misspelling of "JAVA_HOME"
```
Comment (mernst) [inline src/scripts/merge_tools/merge_plumelib.sh:13]: Missing `"` after `JAVA17_HOME`.

**Review by mernst — APPROVED** [review]
This looks good.  Thanks!

**Pull request merged**